hack/create.sh

```diff
@@ -12,10 +12,13 @@ mkdir -p _data/_out
 
 set -x
 
+go generate ./...
+
 USE_PROD_FLAG="-use-prod=false"
 if [[ -n "$TEST_IN_PRODUCTION" ]]; then
     USE_PROD_FLAG="-use-prod=true"
+else
+	go run cmd/fakerp/main.go &
```

Comment:
should probably point to `hack/fakerp.sh`

Reply:
done